modules/microshift-data-cleaning-container-images.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * microshift_troubleshooting/microshift-cleanup-data.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-data-cleaning-container-images_{context}"]
+= Cleaning all data and keeping the container images
+
+Retain the {microshift-short} container images while cleaning all data by running the `microshift-cleanup-data` script with the `--all` and `--keep-images` arguments.
+
+Keeping the container images helps speed up {microshift-short} restart after data cleanup because the necessary container images are already present locally when you start the service.
+
+When you run the script with the `--all` and `--keep-images` arguments, you perform the following cleanup actions:
+
+* All {microshift-short} services are stopped and disabled
+* Deletes all {microshift-short} pods
+* Resets network configuration
+* Deletes the `/var/lib/microshift` data directory
+* Deletes OVN-K networking configuration
```

Comment:
Same comment as the other module about this being a list of actions that "you" the user do, up to you if and how you'd want to change this:
```suggestion
* Stop and disable all {microshift-short} services
* Delete all {microshift-short} pods
* Reset network configuration
* Delete the `/var/lib/microshift` data directory
* Delete OVN-K networking configuration
```